src/main/java/com/google/sps/data/Term.java

```diff
@@ -0,0 +1,64 @@
+package com.google.sps.data;
+
+import java.util.*;
+
+public class Term {
+  String season;
+  int year;
+  String school;
+  ListIterator termIterator;
+  int numTerms;
+  int currTerm;
+  ArrayList<String> termList;
+
+  final List<String> QTR_SCHOOLS = Arrays.asList("caltech", "calpoly", "stanford", "berkeley");
+  final ArrayList<String> QTR_TERMS =
+      new ArrayList<String>(Arrays.asList("Winter", "Spring", "Fall"));
+  final ArrayList<String> SEMESTER_TERMS = new ArrayList<String>(Arrays.asList("Spring", "Fall"));
+  // TODO: Potentially add in a summer term.
+
+  public Term(String termString, String termSchool) {
+    season = termString.split(" ")[0];
+    year = Integer.parseInt(termString.split(" ")[1]);
+    school = termSchool;
+    if (QTR_SCHOOLS.contains(school)) {
```

Comment:
QTR_SCHOOLS is a final, but what is new schools are added to the system?

Reply:
This is just for testing - I'm going to find a list of all quarter system schools and put them there